scripts/globals/job_utils/dark_knight.lua

```diff
@@ -87,36 +90,49 @@ end
 xi.job_utils.dark_knight.useDarkSeal = function(player, target, ability)
     -- Power: Each merit level after the first reduces Dark Magic casting time by -10% (total of -40% bonus).
     -- Sub Power: Enhances Dark Seal effect by increasing duration of Dark Magic by 10% per merit level (total of 50% bonus).
-    local power    = player:getMerit(xi.merit.DARK_SEAL) - 10
-    local subPower = (player:getMerit(xi.merit.DARK_SEAL) / 10) * player:getMod(xi.mod.ENHANCES_DARK_SEAL)
+    local power    = utils.clamp(player:getMerit(xi.merit.DARK_SEAL) - 10, 0, 40)
+    local subPower = utils.clamp(player:getMod(xi.mod.ENHANCES_DARK_SEAL) * player:getMerit(xi.merit.DARK_SEAL), 0, 50)
 
     player:addStatusEffect(xi.effect.DARK_SEAL, power, 0, 60, 0, subPower)
 end
 
 xi.job_utils.dark_knight.useDiabolicEye = function(player, target, ability)
-    player:addStatusEffect(xi.effect.DIABOLIC_EYE, player:getMerit(xi.merit.DIABOLIC_EYE), 0, 180)
+    local power    = 15 + 5 * player:getMerit(xi.merit.DIABOLIC_EYE)
+    local duration = 180 + player:getMod(xi.mod.ENHANCES_DIABOLIC_EYE) * player:getMerit(xi.merit.DIABOLIC_EYE)
+
+    player:addStatusEffect(xi.effect.DIABOLIC_EYE, power, 0, duration)
 end
 
 xi.job_utils.dark_knight.useLastResort = function(player, target, ability)
     player:addStatusEffect(xi.effect.LAST_RESORT, 0, 0, 180)
 end
 
 xi.job_utils.dark_knight.useNetherVoid = function(player, target, ability)
-    player:addStatusEffect(xi.effect.NETHER_VOID, 8, 1, 30)
+    local power = 50 + player:getMod(xi.mod.ENHANCES_NETHER_VOID) + 2 * player:getJobPointLevel(xi.jp.NETHER_VOID_EFFECT)
+
+    player:addStatusEffect(xi.effect.NETHER_VOID, power, 0, 60)
 end
 
 xi.job_utils.dark_knight.useScarletDelirium = function(player, target, ability)
-    player:addStatusEffect(xi.effect.SCARLET_DELIRIUM, 8, 1, 90)
+    local jpValue  = player:getJobPointLevel(xi.jp.SCARLET_DLRIUM_DURATION)
```

Comment:
`DLRIUM`, typo?

Reply:
Nope, although I though it was at first myself.
![imagen](https://user-images.githubusercontent.com/60053999/212740924-eb9d0faf-0d7d-4158-a829-9b74a2b5257c.png)
I could change it in a later PR though. Out of scope for this one.

Reply:
yup save it for later